src/elements/content-preview/stories/ContentPreview.stories.js

```diff
@@ -1,116 +1,66 @@
 // @flow
 import * as React from 'react';
-import { text } from '@storybook/addon-knobs';
-import { IntlProvider } from 'react-intl';
 import ContentPreview from '../ContentPreview';
-import notes from './ContentPreview.notes.md';
 
-export const Preview = () => {
-    const fileId = text('File ID', global.FILE_ID);
-    const token = text('Access Token', global.TOKEN);
+export const Basic = {};
 
-    return (
-        <IntlProvider locale="en">
-            <ContentPreview
-                features={global.FEATURES}
-                fileId={fileId}
-                hasHeader
-                key={`${fileId}-${token}`}
-                token={token}
-            />
-        </IntlProvider>
-    );
-};
-
-export const PreviewWithAnnotations = () => {
-    const fileId = text('File ID', global.FILE_ID);
-    const token = text('Access Token', global.TOKEN);
-
-    return (
-        <IntlProvider locale="en">
-            <ContentPreview
-                contentSidebarProps={{
-                    detailsSidebarProps: {
-                        hasAccessStats: true,
-                        hasClassification: true,
-                        hasNotices: true,
-                        hasProperties: true,
-                        hasRetentionPolicy: true,
-                        hasVersions: true,
-                    },
-                    features: global.FEATURES,
-                    hasActivityFeed: true,
-                    hasMetadata: true,
-                    hasSkills: true,
-                    hasVersions: true,
-                }}
-                features={global.FEATURES}
-                fileId={fileId}
-                hasHeader
-                key={`${fileId}-${token}`}
-                showAnnotations
-                token={token}
-            />
-        </IntlProvider>
-    );
+export const PreviewWithAnnotations = {
```

Comment:
minor consistency thing:
the ContentExplorer and ContentSharing stories drop the element name: `withAnnotations` and `withSidebar`

another consistency thing:
pascal vs camel casing for the story names between Explorer and Sharing